hbase_storage/pom.xml

```diff
@@ -25,148 +25,46 @@
         <dependency>
             <groupId>com.splicemachine</groupId>
             <artifactId>pipeline_api</artifactId>
-            <version>2.8.0.1834-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
does @perryluopl have a python script which needs to change as a result of this edit? I like it the change, but if a script exists he should be aware before he builds this.

Reply:
Perry's script updates all occurrences of the current version, say 2.8.0.1834-SNAPSHOT, to next one in all pom.xml files. It should continue working correctly after the change.
One explicit version per pom.xml is sufficient, I believe.